doc/releases/v0_11.md

```diff
@@ -12,8 +12,13 @@ Important Changes
 #### `sensors`
 * The `iDynTree/Sensors/SixAxisFTSensor.h` header has been deprecated in favor of the  `iDynTree/Sensors/SixAxisForceTorqueSensor.h`.
 
-### `model_io`
+#### `model_io`
 * Added dependency on [Gnome libxml2](http://xmlsoft.org), see next bullet point.
 * Added library `idyntree-modelio-xml` to parse generic XML files. Support for XSD validation (when parsing from file). This library requires [Gnome libxml2](http://xmlsoft.org).
 * Rewritten URDF parser to use the new `idyntree-modelio-xml` library.
 * Dropped required dependency on TinyXML1. If TinyXML is found in the system, it is possible to compile the old URDF parser that has been renamed into `idyntree-modelio-urdf-legacy`.
+
+#### `core`
+* Added `toEigen` methods for `Span` and added typedef for `Eigen` maps of vectors.
+* Added some typedefs to `VectorDynsize` and `VectorFixSize` classes to enable the use of `make_span` with these objects.
+* Added copy operator in `VectorDynSize` and `VectorFixSize` for `Span<double>`.
```

Comment:
@francesco-romano I don't know if this can be helpful for the problem of assigning a position (not direct I think)

Reply:
An `iDynTree::position` you mean? :sweat_smile:

Reply:
Yes.

Reply:
It was possible also by using ``toEigen``, was it?

Reply:
@francesco-romano I think we  just found out why we never implemented the copy constructor :D